src/request_handler.cc

```diff
@@ -65,3 +66,27 @@ void RequestHandler::splitUrl(const char* url, char separator, std::string& path
         path = url_s.substr(0, i1);
     }
 }
+
+std::map<std::string, std::string> RequestHandler::getParameters(const char* filename, const char* baseLink)
```

Comment:
parseParameters?

I always think of getters as simple accessors.

Reply:
good idea